recipes/sdl/all/conanfile.py

```diff
@@ -360,12 +366,12 @@ def package(self):
             self.copy(pattern="COPYING.txt", dst="licenses", src=self._source_subfolder)
         cmake = self._configure_cmake()
         cmake.install()
-        tools.remove_files_by_mask(os.path.join(self.package_folder, "bin"), "sdl2-config")
-        tools.rmdir(os.path.join(self.package_folder, "cmake"))
-        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
-        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
-        tools.rmdir(os.path.join(self.package_folder, "libdata"))
-        tools.rmdir(os.path.join(self.package_folder, "share"))
+        rm(self, os.path.join(self.package_folder, "bin"), "sdl2-config")
```

Comment:
```suggestion
        rm(self, "sdl2-config", os.path.join(self.package_folder, "bin"))
```
It's inverted: https://docs.conan.io/en/latest/reference/conanfile/tools/files/basic.html#conan-tools-files-rm